README.md

```diff
@@ -18,8 +18,35 @@ MiniModelarDB intentionally does not gather usage data. So, all users are highly
 encouraged to post comments, suggestions, and bugs as GitHub issues, especially
 if a limitation of MiniModelarDB prevents it from being used in a particular domain.
 
+## Installation
```

Comment:
"Building" or "Installation"?

Does `cargo build` also place the executable in a specific directory and add configuration?

Reply:
Kept **Installation** as the headline and instead extended **All** to also include how to run the system and how to move the binaries to whatever directory the user prefers.